src/main/java/com/adyen/model/checkout/PaymentMethodsRequest.java

```diff
@@ -87,6 +87,9 @@
     @SerializedName("threeDSAuthenticationOnly")
     private Boolean threeDSAuthenticationOnly = null;
 
+    @SerializedName("store")
+    private String store = null;
```

Comment:
Please update equals() and toString() methods to add this field

Reply:
Nice catch, updated.
Also, the applicationInfo serialised annotation was mispelled, updated that as well.